pkgs/by-name/no/noita-entangled-worlds/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  makeWrapper,
+  steam-run,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "noita_entangled_worlds";
+  version = "0.32.5";
+
+  src = fetchzip {
+    url = "https://github.com/IntQuant/noita_entangled_worlds/releases/download/v${finalAttrs.version}/noita-proxy-linux.zip";
+    sha256 = "sha256-qvcSf82RqPoRVwVVWqMcQNEa03b/asLgBwD5hsbyBoE=";
+    stripRoot = false;
+  };
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+  buildInputs = [
+    steam-run
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    mkdir -p $out/libexec
+    cp noita_proxy.x86_64 $out/libexec/noita-proxy-bin
+    cp libsteam_api.so $out/libexec/
+
+    # Create a wrapper script that calls the binary from libexec
+    makeWrapper ${steam-run}/bin/steam-run $out/bin/noita-proxy \
+      --add-flags "$out/libexec/noita-proxy-bin"
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    changelog = "https://github.com/IntQuant/noita_entangled_worlds/releases/tag/v${finalAttrs.version}";
+    description = "Noita Entangled Worlds - Prebuilt binary for Linux";
+    homepage = "https://github.com/IntQuant/noita_entangled_worlds";
+    license = licenses.mit;
```

```suggestion
    license = with lib.licenses; [ asl20 mit ];
```
https://github.com/IntQuant/noita_entangled_worlds/blob/master/LICENSE-APACHE
https://github.com/IntQuant/noita_entangled_worlds/commit/13dcb9da6fce94709a5cf95928a13eeb9265452a